selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/GolangFileEditingTest.java

```diff
@@ -153,4 +190,151 @@ public void checkFindDefinitionFeature() {
     editor.waitCursorPosition(24, 6);
     editor.clickOnCloseFileIcon("print.go");
   }
+
+  @Test(priority = 1)
+  public void checkRenameCodeFeature() {
```

Comment:
Test method sounds a little bit strange: it is not about renaming the code - it is about renaming something inside the code, which is part of code.
So, I would propose to rename the test to `checkRenameFeature`

Reply:
OK, agree